### DISCUSSION TITLE
Edit group with unknown members

### DISCUSSION THREAD

**Discussion by tjorim** [frenck/spook#590] [Ideas]
When Spook finds a broken automation, it provides a direct link to edit that automation. When Spook finds a broken group, it does not. It would be useful that the repair also includes a link to edit the group options. ![Screenshot_20240208-101001.png](https://github.com/frenck/spook/assets/1294876/0846cfcc-3428-46b1-8806-74135e72df50)

**Comment by frenck** [comment]
I don't think that is possible?

What would that link look like?

**Reply by tjorim** [reply]
I see what you mean, it's not a `url` but just a pop-up. ![Screenshot_20240208-111917.png](https://github.com/frenck/spook/assets/1294876/6a85496f-1b11-4eb7-ab17-bcee87f7ab4b)

In case it's not possible, even a link to the helpers page would already save us some clicks: https://my.home-assistant.io/redirect/helpers/
